### PR TITLE
Add docker-volume-ipfs plugin to the list.

### PR DESCRIPTION
**Pull request by vdemeester**
:angel: 🐸

Signed-off-by: Vincent Demeester vincent@sbr.pm

### CODE REVIEW

File: docs/extend/plugins.md [hunk 1]
```diff
@@ -67,16 +67,25 @@ The following plugins exist:
   platforms including VirtualBox, EC2, Google Compute Engine, OpenStack, and EMC.
 
 * The [Contiv Volume Plugin](https://github.com/contiv/volplugin) is an open
-source volume plugin that provides multi-tenant, persistent, distributed storage
-with intent based consumption using ceph underneath.
+  source volume plugin that provides multi-tenant, persistent, distributed storage
+  with intent based consumption using ceph underneath.
```
Comment (thaJeztah) [inline docs/extend/plugins.md]: `<3`

File: docs/extend/plugins.md [hunk 2]
```diff
@@ -67,16 +67,25 @@ The following plugins exist:
...
+
+* The [Weave Network Plugin](http://docs.weave.works/weave/latest_release/plugin.html)
+  creates a virtual network that connects your Docker containers -
+  across multiple hosts or clouds and enables automatic discovery of
+  applications. Weave networks are resilient, partition tolerant,
+  secure and work in partially connected networks, and other adverse
+  environments - all configured with delightful simplicity. 
+
+* The [IPFS Volume Plugin](http://github.com/vdemeester/docker-volume-ipfs)
+  is an open source volume plugin than provide access to
```
Comment (thaJeztah) [inline docs/extend/plugins.md]: s/provide/provides/

File: docs/extend/plugins.md [hunk 3]
```diff
@@ -67,16 +67,25 @@ The following plugins exist:
...
+  [ipfs](https://ipfs.io/) filesystem as volume.
```
Comment (thaJeztah) [inline docs/extend/plugins.md]: perhaps;

"The IPFS volume plugin is an open source volume plugin that allows using a ipfs filesystem as a volume."

Comment (vdemeester) [inline docs/extend/plugins.md]: Make sence :stuck_out_tongue_closed_eyes:

**Comment by thaJeztah** [comment]
Thanks, LGTM!

**Pull request merged**